tests/virt/conftest.py

```diff
@@ -145,3 +155,65 @@ def vm_cpu_flags(nodes_cpu_virt_extension):
         if nodes_cpu_virt_extension
         else None
     )
+
+
+@pytest.fixture(scope="session")
+def supported_gpu_device(workers_utility_pods, nodes_with_supported_gpus):
+    gpu_info = get_nodes_gpu_info(util_pods=workers_utility_pods, node=nodes_with_supported_gpus[0])
+    for gpu_id in GPU_CARDS_MAP:
+        if gpu_id in gpu_info:
+            return GPU_CARDS_MAP[gpu_id]
+
+    raise ResourceNotFoundError("GPU device ID not in current GPU_CARDS_MAP!")
```

Comment:
This exception is not suitable as it is for k8s resources not found. please create a new exception for this use case